e2e/config/config.go

```diff
@@ -145,7 +145,7 @@ func readConfig(configFile string, config *Config) error {
 	viper.SetDefault("Repo.URL", defaultGitURL)
 	viper.SetDefault("Repo.Branch", defaultGitBranch)
 	viper.SetDefault("DRPolicy", defaultDRPolicyName)
-	viper.SetDefault("ClusterSet", defaultClusterSetName)
+	viper.SetDefault("ClusterSet", DefaultClusterSetName)
```

Comment:
Why DefaultClusterSetName starts with capital letter and others with small?

Reply:
We need it to create the generic ramenctl config. We can keep this private and add the same config in ramenctl, or make this public and access it in ramenctl.

The other defaults are related only to testing so ramenctl do not need them.